Templates loading menus

armandohimself opened the pull request:
**What Was Done**
1. Conditional render for draft and publish in the events tab under event templates
2. Fix menus loading in event templates

armandohimself committed:
Only Show Publishing Events in Your Events

As a chef
I should only see draft and publish in Your Events
So I'm not confused when adding a templated event to my events lists. 

- Draft and publish only show up in your events tab when expanding on more details.

armandohimself committed:
Fix Menus Showing in Events Tab

Users can see menus from event templates again.

armandohimself committed:
Accidentally Left Console Logs

commented out consoles.

JoshuaArcyLHarrison reviewed (commented):
This is fine for now, but think we should separate these into two different screens.

armandohimself committed:
Console.logs

Commit:
Filters on published and current events

Co-authored-by: Armando Arteaga <hey@armandohimself.com>

armandohimself committed:
Merge Main

armandohimself committed:
Format Profile Slider & Add Back Button

- Added back button to login screen
- Formatted profile slider

armandohimself committed:
Fixed Complete Chef Profile

What Was Done
- Fix Conditional Rendering of Card on Home Screen
- Fix Initial Chef Object when registering 
- Fix Cuisine Array to hold all selections made
- Refactor Chef Profile Object in Chef Object
- Validate that data gets updated in FireStore

armandohimself committed:
Merge branch 'completeChefProfile' into templatesLoadingMenus

armandohimself committed:
Pushing Changes

Commit:
Fix create event screen and update data variables

Commit:
Update event data model everywhere

Commit:
BROKEN: Updating ChefEventDetail to use new model

Commit:
Fix event detail screen

armandohimself committed:
Fix Chef Event Screen

- Cleaned up some of the data pulling event templates
- Moved file to appropriate folders
- Updated navigation to reflect files moving around
- Added helper to sort an object returned from service based on property in the object you give it.
- Moved styling out of event screen
- Moved custom circle button to CustomButton component
- Moved empty image state to component

armandohimself committed:
Fix Chef Event Screen

- Cleaned up some of the data pulling event templates
- Moved file to appropriate folders
- Updated navigation to reflect files moving around
- Added helper to sort an object returned from service based on property in the object you give it.
- Moved styling out of event screen
- Moved custom circle button to CustomButton component

armandohimself committed:
Merge remote-tracking branch 'origin/templatesLoadingMenus' into templatesLoadingMenus

armandohimself committed:
Moved Chef Files Around Part 2

- Create Event Screen and Event Listings Component moved to chef folders

Commit:
Merge branch 'main' into templatesLoadingMenus
